src/design-mode-component.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+function DesignModeComponent() {
+  this.state = false;
+  chrome.runtime.onMessage.addListener(function (request) {
+    if (request.trigger == 'toggleDesignMode') {
```

Comment:
Everywhere else in the file is `"`, can we make this double quotes too? (And the `"use strict"`)